select_statements.sql

```diff
@@ -11,16 +11,16 @@
 
 -- Uppdatera bokning med nytt datum
 UPDATE booking
-SET endDatum = "2024-05-06"
+SET endDate = "2024-05-06"
 WHERE stationName = "Uppsala station"
 AND personalIdentificationNumber = "900530-1234"
-AND startDatum = "2024-05-04";
+AND startDate = "2024-05-04";
```

Comment:
I realize that you didn't create this originally, but here I think we need to add the id field as a predicate, because you could have two bookings with the same startDate.